packages/paste-core/components/form/__tests__/index.spec.tsx

```diff
@@ -0,0 +1,179 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {
+  Form,
+  FormActions,
+  FormControl,
+  FormControlTwoColumn,
+  FormSection,
+  FormSectionDescription,
+  FormSectionHeading,
+} from '../src';
+
+describe('Form', () => {
+  it('should render correctly', () => {
+    render(
+      <Form aria-label="Settings">
+        <FormSection id="foo">
+          <FormSectionHeading data-testid="section-heading">Settings</FormSectionHeading>
+          <FormSectionDescription data-testid="section-desc">These are the settings</FormSectionDescription>
+        </FormSection>
+        <FormControlTwoColumn>Two column content</FormControlTwoColumn>
+        <FormControl>Control content</FormControl>
+        <FormActions>Actions content</FormActions>
+      </Form>
+    );
+
+    const form = screen.getByRole('form');
+    expect(form).toBeDefined();
+    expect(form).toHaveAttribute('aria-label', 'Settings');
+
+    const description = screen.getByTestId('section-desc');
+    expect(description.id).toBe('foo-section-description');
+
+    const section = screen.getByRole('group', {name: 'Settings'});
```

Comment:
question: this was unclear to me, but maybe it's from a lack of familiarity w/ `group`- is the name of this group 'Settings' because of the content of `<FormSectionHeading>` or because of `aria-label="Settings"` on `Form`?

Reply:
This is because the fieldset element (which is rendered by FormSection) has role="group" implicitly and then when a fieldset has a legend (which is rendered by FormSectionHeading) as the first child, that legend is automatically assumed to be the name of the group.

To clear up confusion, I changed the Form's aria-label attribute to be "My Form" instead of also being "settings".